src/pages/LandingPage.js

```diff
@@ -30,7 +30,7 @@ class Home extends Component {
     event.preventDefault();
 
     if (this.state.username.trim() !== "") {
-      this.props.history.push("/user/" + this.state.username + "/resume");
+      this.props.history.push("/user/" + this.state.username.toLowerCase() + "/resume");
```

Comment:
Do all Github usernames have lower case letters?

Reply:
Sorry, thought that usernames are lowercase. Wait will change it

Reply:
Github usernames are case insensitive. so no need of .toLowerCase() function
 
https://stackoverflow.com/questions/14712475/will-changing-my-github-username-to-snakecase-break-anything.

Reply:
@sabesansathananthan okay